docker/1.14/Dockerfile.eia

```diff
@@ -0,0 +1,91 @@
+FROM ubuntu:16.04
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ARG TFS_SHORT_VERSION=1.14
+ARG TFS_VERSION=1-14-0
+ARG EI_VERSION=1-2
+ARG PYTHON=python3
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.6.6
+ARG HEALTH_CHECK_VERSION=1.3.3
+
+# nginx + njs
+RUN apt-get update && \
+    apt-get -y install --no-install-recommends \
+    curl \
+    gnupg2 \
+    ca-certificates \
+    git \
+    wget \
+    vim \
+    build-essential \
+    zlib1g-dev && \
+    curl -s http://nginx.org/keys/nginx_signing.key | apt-key add - && \
+    echo 'deb http://nginx.org/packages/ubuntu/ xenial nginx' >> /etc/apt/sources.list && \
+    apt-get update && \
+    apt-get -y install --no-install-recommends nginx wget nginx-module-njs  && \
+    apt-get clean 
+
+# See http://bugs.python.org/issue19846
+ENV LANG C.UTF-8
+
+RUN wget https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tgz && \
+    tar -xvf Python-$PYTHON_VERSION.tgz && cd Python-$PYTHON_VERSION && \
+    ./configure && make && make install && \
+    apt-get update && apt-get install -y --no-install-recommends libreadline-gplv2-dev libncursesw5-dev libssl-dev libsqlite3-dev tk-dev libgdbm-dev libc6-dev libbz2-dev && \
+    make && make install && rm -rf ../Python-$PYTHON_VERSION* && \
+    ln -s /usr/local/bin/pip3 /usr/bin/pip
+
+# Python won’t try to write .pyc or .pyo files on the import of source modules
+ENV PYTHONDONTWRITEBYTECODE=1 PYTHONUNBUFFERED=1
+
+# cython, falcon, gunicorn
+RUN ${PIP} install cython falcon gunicorn gevent requests 
+
+COPY sagemaker /sagemaker
+
+RUN wget https://amazonei-healthcheck.s3.amazonaws.com/v${HEALTH_CHECK_VERSION}/ei_health_check_${HEALTH_CHECK_VERSION}.tar.gz -O /opt/ei_health_check_${HEALTH_CHECK_VERSION}.tar.gz \
+    && tar -xvf /opt/ei_health_check_${HEALTH_CHECK_VERSION}.tar.gz -C /opt/ \
+    && rm -rf /opt/ei_health_check_${HEALTH_CHECK_VERSION}.tar.gz \
+    && chmod a+x /opt/ei_health_check/bin/health_check
+
+RUN ${PIP} --no-cache-dir install  --upgrade pip setuptools 
+
+# Some TF tools expect a "python" binary
+RUN ln -s $(which ${PYTHON}) /usr/local/bin/python &&\
+    ${PIP} install -U --no-cache-dir docutils==0.14 awscli==1.16.196
+
+# Expose ports
+# gRPC
+EXPOSE 8500
+
+# REST
+EXPOSE 8501
+
+RUN wget https://amazonei-tensorflow.s3.amazonaws.com/tensorflow-serving/v1.14/ubuntu/archive/tensorflow-serving-${TFS_VERSION}-ubuntu-ei-${EI_VERSION}.tar.gz -O /tmp/tensorflow-serving-${TFS_VERSION}.tar.gz
+
+RUN cd /tmp && \
+    tar zxf tensorflow-serving-${TFS_VERSION}.tar.gz && \
+    rm -f tensorflow-serving-${TFS_VERSION}.tar.gz && \
```

Comment:
remove this since we are already cleaning up in the end

Reply:
This clean up specifically removes the tar file for tensorflow-serving. The line after removing the tar file ensures that the name of the created directory is always known (by renaming the dir), but this requires that the tar file be deleted beforehand.

Reply:
Removed and made the necessary changes.